xmlserializer/CMakeLists.txt

```diff
@@ -34,6 +34,8 @@ add_library(xmlserializer SHARED
     XmlMemoryDocSource.cpp
     XmlStreamDocSink.cpp)
 
+set_target_properties(xmlserializer PROPERTIES POSITION_INDEPENDENT_CODE TRUE)
+
```

Comment:
That is great ! Should use it for [utilty](https://github.com/01org/parameter-framework/blob/master/utility/CMakeLists.txt#L36)